serviceworker.js

```diff
@@ -11,38 +11,34 @@ importScripts('reconstructive.js');
 //   showBanner: false,
 //   debug: false
 // });
-let currentPath = self.location.href.substring(0, self.location.href.lastIndexOf('/'));
-Reconstructive.init({
+const currentPath = self.location.href.substring(0, self.location.href.lastIndexOf('/'));
+const reconstructive = new Reconstructive({
   debug: true,
   showBanner: true,
   bannerElementLocation: `${currentPath}/reconstructive-banner.js`,
-  urimPattern: `${currentPath}/tests/<datetime>/<urir>`
+  urimPattern: `${currentPath}/tests/<datetime>/<urir>`,
```

Comment:
Remove this trailing comma to consistent.

Reply:
done